research/delf/delf/python/datasets/tuples_dataset.py

```diff
@@ -0,0 +1,353 @@
+# Lint as: python3
+# Copyright 2021 The TensorFlow Authors All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tuple dataset based on the Radenovic et al. ECCV16: CNN image retrieval
+learns from BoW.
+
+For more information refer to https://arxiv.org/abs/1604.02426.
+"""
+
+from absl import flags
+
+import os
+import pickle
+import tensorflow as tf
+import numpy as np
+import matplotlib.pyplot as plt
+
+from delf.python.datasets import utils as image_loading_utils
+from delf.python.datasets import generic_dataset
+from delf.python.training import global_features_utils
```

Comment:
Can you please reorder these imports as follows:

import os
import pickle

from absl import flags
import matplotlib.pyplot as plt
import numpy as np
import tensorflow as tf

from delf.python.datasets import generic_dataset
from delf.python.datasets import utils as image_loading_utils
from delf.python.training import global_features_utils

Reply:
Thank you, fixed